.github/workflows/update-engines-version.yml

```diff
@@ -42,6 +42,14 @@ jobs:
             pnpm update -r @prisma/engines@${{ github.event.inputs.version }} @prisma/engines-version@${{ github.event.inputs.version }} @prisma/fetch-engine@${{ github.event.inputs.version }} @prisma/get-platform@${{ github.event.inputs.version }}
             echo 'Awesome - proceeding to make the PR'
 
+      - name: Extract Engine Commit hash from version
+        id: extract-engine-commit-hash
+        uses: actions/github-script@v6
```

Comment:
Someone found the all you can script GitHub action 😉

Reply:
Indeed - isn't it amazing!? Much better than doing some weird bash thingie.

Reply:
YEss!